### ISSUE TITLE
[1.18.2] Log spam

### ISSUE THREAD

**Issue by jeremiahwinsley** [MehVahdJukaar/Supplementaries#221]
During startup there is a lot of log spam from Supplementaries:

Modpack: ATM7 0.3.0
Supplementaries: 1.3.2

latest.log: https://gist.github.com/jeremiahwinsley/6fecd06028b4734b627c926b4627a520

examples:

```
[01Apr2022 17:54:29.183] [Render thread/ERROR] [net.mehvahdjukaar.supplementaries.Supplementaries/]: Failed to generate Hanging Sign block texture for for Block{supplementaries:securitycraft/hanging_sign_reinforced_oak} : java.io.FileNotFoundException: Could not find any texture associated to the given block securitycraft:reinforced_oak_planks
```

```
[01Apr2022 17:54:29.217] [Render thread/ERROR] [net.mehvahdjukaar.supplementaries.Supplementaries/]: Could not find sign texture for wood type byg:aspen. Using plank texture : java.lang.NullPointerException: Cannot invoke "net.minecraft.world.item.Item.getRegistryName()" because "item" is null
```

```
[01Apr2022 17:54:45.649] [Worker-Main-7/ERROR] [net.minecraft.client.renderer.texture.TextureAtlas/]: Using missing texture, unable to load supplementaries:textures/entity/sign_posts/securitycraft/sign_post_reinforced_acacia.png : java.io.FileNotFoundException: supplementaries:textures/entity/sign_posts/securitycraft/sign_post_reinforced_acacia.png
[01Apr2022 17:54:45.650] [Worker-Main-4/ERROR] [net.minecraft.client.renderer.texture.TextureAtlas/]: Using missing texture, unable to load supplementaries:textures/blocks/amethyst_block.png : java.io.FileNotFoundException: supplementaries:textures/blocks/amethyst_block.png
```

**Comment by MehVahdJukaar** [comment]
That was caused by my dynamic wood system wrongfully detecting securitycraft stuff as a wood type (it fools it since it both has a plank and a log type which exibit the property of a proper wood type)
I've added a special case to make it not include it so it shouldnt be there anymore

**Comment by jeremiahwinsley** [comment]
Sorry if I wasn't clear, there was more than just the SecurityCraft stuff. After updating to 1.3.3, the SecurityCraft entries are gone, but most of the rest remains. Links to specific lines:

https://gist.github.com/jeremiahwinsley/6fecd06028b4734b627c926b4627a520#file-latest-log-L371-L462
https://gist.github.com/jeremiahwinsley/6fecd06028b4734b627c926b4627a520#file-latest-log-L838-L843
https://gist.github.com/jeremiahwinsley/6fecd06028b4734b627c926b4627a520#file-latest-log-L907-L915

Not sure if this is just more special case stuff or other cause.

**Comment by MehVahdJukaar** [comment]
I'll just remove it so it wont log anything. It's just complaining for all the mods that add a wood type but dont add a vanilla sign item (like byg) since it searches for it when making my signs item textures since they are based off it. still works since it fallsback to the wood one. I guess it's good for debugging but can be annoying

**Comment by MehVahdJukaar** [comment]
Should be fixed in 1.18.2

**Comment by Maypul** [comment]
@MehVahdJukaar I will add that there appears to be still some spam in the log, in case it can be somehow useful:

[19:49:49] [Worker-Main-12/WARN] [minecraft/ModelBakery]: Unable to load model: 'minecraft:item/crossbow/crossbow_bolt_iron' referenced from: minecraft:crossbow#inventory: java.io.FileNotFoundException: minecraft:models/item/crossbow/crossbow_bolt_iron.json
[19:49:49] [Worker-Main-12/WARN] [minecraft/ModelBakery]: Unable to load model: 'minecraft:item/crossbow/crossbow_bolt_netherite' referenced from: minecraft:crossbow#inventory: java.io.FileNotFoundException: minecraft:models/item/crossbow/crossbow_bolt_netherite.json
[19:49:49] [Worker-Main-12/WARN] [minecraft/ModelBakery]: Unable to load model: 'minecraft:item/crossbow/crossbow_bolt_diamond' referenced from: minecraft:crossbow#inventory: java.io.FileNotFoundException: minecraft:models/item/crossbow/crossbow_bolt_diamond.json
[19:49:49] [Worker-Main-12/WARN] [minecraft/ModelBakery]: Unable to load model: 'minecraft:item/crossbow/crossbow_bolt_gold' referenced from: minecraft:crossbow#inventory: java.io.FileNotFoundException: minecraft:models/item/crossbow/crossbow_bolt_gold.json
[19:49:49] [Worker-Main-12/WARN] [minecraft/ModelBakery]: Unable to load model: 'supplementaries:blocks/wall_lanterns/wall_lantern_copper' referenced from: supplementaries:blocks/wall_lanterns/wall_lantern_copper: java.io.FileNotFoundException: supplementaries:models/blocks/wall_lanterns/wall_lantern_copper.json
[19:49:49] [Worker-Main-12/WARN] [minecraft/ModelBakery]: Unable to load model: 'supplementaries:blocks/wall_lanterns/wall_lantern_brass' referenced from: supplementaries:blocks/wall_lanterns/wall_lantern_brass: java.io.FileNotFoundException: supplementaries:models/blocks/wall_lanterns/wall_lantern_brass.json
[19:49:49] [Worker-Main-12/WARN] [minecraft/ModelBakery]: Unable to load model: 'supplementaries:blocks/wall_lanterns/wall_lantern_gold' referenced from: supplementaries:blocks/wall_lanterns/wall_lantern_gold: java.io.FileNotFoundException: supplementaries:models/blocks/wall_lanterns/wall_lantern_gold.json
[19:50:01] [Worker-Main-14/ERROR] [minecraft/TextureAtlas]: Using missing texture, unable to load supplementaries:textures/blocks/wall_lantern.png : java.io.FileNotFoundException: supplementaries:textures/blocks/wall_lantern.png
[19:50:01] [Worker-Main-3/ERROR] [minecraft/TextureAtlas]: Using missing texture, unable to load supplementaries:textures/block/flower_pot.png : java.io.FileNotFoundException: supplementaries:textures/block/flower_pot.png
